plugins/advisors/black-duck/src/main/kotlin/BlackDuck.kt

```diff
@@ -220,6 +224,18 @@ internal fun VulnerabilityView.toOrtVulnerability(): Vulnerability {
     )
 }
 
+private fun VulnerabilityCvss3View.getScoringSystemAndVector(): Pair<String?, String?> {
```

Comment:
Looks like the return string actually can never be `null`, so the signature should be adjusted. Similar for the other function.